Projects/Server/Utilities/Random.cs

```diff
@@ -22,12 +22,11 @@ namespace Server
   public static class RandomImpl
   {
     private static readonly IRandomImpl _Random;
+    // private static readonly IRandomImpl _SecureRandom;
 
     static RandomImpl()
     {

```

Comment:
Don't you want to prefer xoroshiro here?

Reply:
I haven't gotten there yet. Technically this PR isn't done. I want to redo how secure RNG is used and test it. You are right though, and I will be doing that!